src/funct.js

```diff
@@ -1,3 +1,27 @@
-export default function printMe() {
-  return 'I get called from print.js!';
-}
\ No newline at end of file
+/* eslint-disable-next-line import/no-cycle */
+import dataLoading from './index.js';
+
+const tasks = JSON.parse(localStorage.getItem('datas'));
+
+const validateForm = (event) => {
+  event.preventDefault();
+  const task = document.getElementById('newTask');
+  if (task && task.value !== '') {
+    const newTask = {
+      index: tasks.length,
+      completed: false,
+      description: task && task.value,
+    };
+    task.value = '';
+
+    tasks.push(newTask);
+    localStorage.setItem('datas', JSON.stringify(tasks));
+
+    dataLoading();
+
+    return false;
```

Comment:
Please note that I keep getting the below error when I try adding a task to the to-do app, I think it has to do with the funct.js file, Kindly look into this 👍🏽
![Screenshot from 2023-05-16 17-23-41](https://github.com/MenendezON/WP-To-Do-list/assets/44624138/1fcaad0c-8448-4ace-affd-55871a0f203f)